src/editor/panels/editor_panel_canvas_properties.cpp

```
@@ -1,28 +1,28 @@
 
 void EditorPanelCanvasProperties::DrawContents() {
-    imgui_ext::InputIntVec2("Canvas Size", &m_editorLayer.GetConfig().CanvasSize);
-    ImGui::InputInt("Canvas Zoom", &m_canvasPanel.m_canvasZoom);
-    m_canvasPanel.m_canvasZoom = std::clamp(m_canvasPanel.m_canvasZoom, s_minZoom, s_maxZoom);
-    ImGui::PushItemWidth(ImMax(1.0f, (ImGui::CalcItemWidth() - 6) / 2.0f));
-    ImGui::InputInt("##Grid Spacing", &m_editorLayer.GetConfig().CanvasGridSpacing);
-    ImGui::SameLine();
-    ImGui::InputInt("Grid Spacing/Major Factor", &m_editorLayer.GetConfig().CanvasGridMajorFactor);
-    ImGui::PopItemWidth();
+    auto& canvasSize = m_editorLayer.GetConfig().CanvasSize;
+    float const separatorW = ImGui::CalcTextSize("x").x;
```

Clang (14.0.6)

[error] variable 'separatorW' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)